Readme.md

```diff
@@ -1,146 +1,151 @@
+  order of **ggplot2** functions.
+
+- **formatR** does not add line breaks after `+`s in **ggplot2** code.
+
+- **styler** reformats long lines by putting each argument on its own
+  line and indenting, for example:
+
+<!-- -->
+
+    ggplot() +
+        geom_ribbon(data = ribbon, aes(
+            ymin = min,
+            ymax = max,
+            x = x.ribbon,
+            fill = 'lightgreen'
+        )) +
+        geom_line(data = ribbon, aes(x = x.ribbon, y = avg, color = 'black')) +
+        geom_line(data = data, aes(x = x, y = new.data, color = 'red')) +
+        scale_fill_identity(name = 'the fill',
+                            guide = 'legend',
+                            labels = c('m1')) +
+        scale_colour_manual(
+            name = 'the colour',
+            values = c('black' = 'black', 'red' = 'red'),
+            labels = c('c2', 'c1')
+        ) +
+        xlab('x') +
+        ylab('density')
+
+**ggformat** styles this code as:
+
+    ggplot() +
+      geom_ribbon(data = ribbon, aes(ymin = min, ymax = max, x = x.ribbon, fill = 'lightgreen')) +
+      geom_line(data = ribbon, aes(x = x.ribbon, y = avg, color = 'black')) +
+      geom_line(data = data, aes(x = x, y = new.data, color = 'red')) +
+      scale_fill_identity(name = 'the fill', guide = 'legend', labels = c('m1')) +
+      scale_colour_manual(name = 'the colour', values = c('black' = 'black', 'red' = 'red'), labels = c('c2', 'c1')) +
+      xlab('x') +
+      ylab('density')
+
+Note: I hope to add wrapping so long lines are broken apart after
+commas:
+
+    # NOT YET IMPLEMENTED
+    ggplot() +
+      geom_ribbon(data = ribbon, aes(ymin = min, ymax = max, x = x.ribbon, fill = 'lightgreen')) +
+      geom_line(data = ribbon, aes(x = x.ribbon, y = avg, color = 'black')) +
+      geom_line(data = data, aes(x = x, y = new.data, color = 'red')) +
+      scale_fill_identity(name = 'the fill', guide = 'legend', labels = c('m1')) +
+      scale_colour_manual(name = 'the colour', values = c('black' = 'black', 'red' = 'red'),
+        labels = c('c2', 'c1')) +
+      xlab('x') +
+      ylab('density')
+
+## Nitty Gritty
+
+- +’s are cleaned up so there is exactly one + at the end of every line
+  except the last, and lines ending in pipes. Specifically, double +’s
+  are replaced with single +’s, +’s before special **ggplot2** words are
+  removed, +’s are added back at the ends of lines.
+
+## Known issues
+
+- Doesn’t wrap long lines
+
+- Can’t handle package namespacing such as `ggalluvial::geom_flow()`
+
+- Can’t handle comments (puts them before `+`)
+
+I welcome your suggestions for improvements.
+
+## Acknowledgements
+
+The blog post [“strsplit – but keeping the
+delimiter”](https://www.statworx.com/de/blog/strsplit-but-keeping-the-delimiter/)
+by Jakob Gepp was helpful for formulating `strsplit()` splits that keep
+the search terms.
+
+## How To : 
+Here's the link of a how to video on installation, usage and contribution : [link](...)

```

Comment:
Once you have the link, please add it here and then I'll merge it (whether or not the namespace issue is completed.)